main.go

```diff
@@ -62,6 +65,13 @@ func main() {
 			Value:       false,
 			Destination: &useDurableStore,
 		}),
+		altsrc.NewBoolFlag(&cli.BoolFlag{
```

Comment:
Completely beyond the scope of this PR (and not really that important) but it might be nice to support some kind of `EnumFlag`.  That would allow for a more descriptive flag, IE:`-peerdiscovery=dht` vs `-peerdiscovery=mdns`

It looks like `urFav` doesn't have an enum flag  out of the box but it [looks pretty easy to implement your own](https://github.com/urfave/cli/issues/602#issuecomment-416000118)